shopping-cart/app/shoppingcart/ErrorBean.java

```diff
@@ -1,4 +1,4 @@
-package purchase;
```

Comment:
I think ErrorBean and ErrorsBean could be in the global common so that error handling is the same for all modules.